templates/registration/logout.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
+{% block content %}
+  <p>Logged out!</p>
+  <a href="{% url 'login'%}">Click here to login again.</a>
```

Comment:
Verify that the URL name 'login' matches the URL pattern defined in your project's `urls.py`. Ensure consistency with the naming conventions used throughout the project.